source/code-snippets/usage-examples/bulkWrite-example.js

```diff
@@ -44,7 +40,7 @@ async function run() {
           console.log("Number of documents inserted: " + result.nInserted);
         } else {
           console.log(
-            "No documents were inserted during the bulk write operation"
+            "No documents were inserted during the bulk write operation",
```

Comment:
Nit: This comma could be confusing for newcomers to our documentation - I'd advise removing it (from all examples)
```suggestion
            "No documents were inserted during the bulk write operation"
```

Reply:
I can do that it's just not the purpose of this PR, the only thing added to this line was the comma (which was added by eslint)

Reply:
Fixed